### PR TITLE
Article refresh: Get access without a user

### PR DESCRIPTION
**Pull request by FaithOmbongi**
Refresh this article + address customer feedback.

- https://github.com/microsoftgraph/microsoft-graph-docs/issues/18769
- https://github.com/microsoftgraph/microsoft-graph-docs/issues/18731
- https://github.com/microsoftgraph/microsoft-graph-docs/issues/20272
- https://github.com/microsoftgraph/microsoft-graph-docs/issues/20350
- https://github.com/microsoftgraph/microsoft-graph-docs/issues/20597
- https://github.com/microsoftgraph/microsoft-graph-docs/issues/20597

### CODE REVIEW

File: concepts/auth-v2-service.md [hunk 1]
```diff
@@ -85,7 +101,7 @@ GET https://login.microsoftonline.com/{tenant}/adminconsent
 
 ### Administrator consent experience
 
-With requests to the `/adminconsent` endpoint, Azure AD enforces that only a tenant administrator can sign in to complete the request. The administrator will be asked to approve all the application permissions that you've requested for your app in the app registration portal.
+With requests to the `/adminconsent` endpoint, Azure AD enforces that only a tenant administrator can sign in to complete the request. The administrator is asked to approve all the application permissions that you've requested for your app in the app registration portal.
```
Comment (Jackson-Woods) [inline concepts/auth-v2-service.md:104]: By "tenant admin" we mean "global admin for the organization", right?

Typically we try to minimize referring to tenants, and we should be more specific about the role required IMO.

**Review by Jackson-Woods — APPROVED** [review]
Looks good to me! Minor suggestions.

File: concepts/auth-v2-service.md [hunk 2]
```diff
@@ -1,73 +1,82 @@
...
 
-Apps that call Microsoft Graph with their own identity use the OAuth 2.0 [client credentials grant flow](/azure/active-directory/develop/v2-oauth2-client-creds-grant-flow) to get access tokens from Azure AD. This article describes the basic steps to configure a service and use the OAuth client credentials grant flow to get an access token.
+This article details the raw HTTP requests involved for an app to call Microsoft Graph with its own identity using a popular flow called the [OAuth 2.0 client credentials grant flow](/azure/active-directory/develop/v2-oauth2-client-creds-grant-flow). A [Microsoft-built and supported authentication library](#use-the-microsoft-authentication-library-msal) handles the authorization and token requests and helps you to get access tokens and call Microsoft Graph.
```
[the remark this answers is not in the document]
Comment (FaithOmbongi) [reply concepts/auth-v2-service.md:17]: Agree and done.

**Comment by FaithOmbongi** [comment]
Hi @Jackson-Woods, @marcusca10 - Thank you both for your well thought feedback. I've applied most of it bar one or two that I'll bring up for further discussion.

**Pull request merged**